ci/infra/testrunner/utils/utils.py

```diff
@@ -65,6 +67,23 @@ def cleanup_files(files):
         for file in files:
             Utils.cleanup_file(file)
 
+    def ssh_cleanup(self):
+        """Remove ssh sock files"""
+        # TODO: also kill ssh agent here? maybe move pkill to kill_ssh_agent()?
```

Comment:
i guess the [WIP] is because of this TODO here? let's follow up on that in a separate PR to unblock upgrade testing

Reply:
That, and because I hadn't validated that it actually ran at the time I created the PR.  So mostly because I forgot to remove the WIP. :D